libdrgn/configure.ac

```diff
@@ -37,6 +37,24 @@ AS_CASE(["x$enable_openmp"],
 AC_SUBST(OPENMP_CFLAGS)
 AC_SUBST(OPENMP_LIBS)
 
+AC_ARG_ENABLE([asan],
+	      [AS_HELP_STRING([--enable-asan@<:@=ARG@:>@],
+			      [use ASAN. ARG may be yes or no.
+			       @<:@default=no@:>@])],
+	      [], [enable_asan=no])
+
+ASAN_CFLAGS=
```

Comment:
Can we name these `SANITIZER_CFLAGS` and `SANITIZER_LDFLAGS` so that we can use them to configure other sanitizers in the future?